vercel.json

```diff
@@ -71,6 +71,14 @@
     {
       "source": "/docs/legal/licensing/",
       "destination": "/docs/legal/licensing-and-trademark"
+    },
+    {
+      "source": "/docs/software/mqtt",
```

Comment:
We should probably consider turning on trailing slash so we don't have to deal with stuff like this and it's better for SEO

https://vercel.com/docs/projects/project-configuration#trailingslash